### ISSUE TITLE
Consider adding functional tests to this project

### ISSUE THREAD

**Issue by danielbachhuber** [wpup/wp-cli-media-restore#1]
Functional tests are an integral ingredient of highly-quality, maintainable commands. WP-CLI tries to make it as easy as possible to add functional tests to your package with its `wp scaffold package-tests` command:

https://github.com/wp-cli/scaffold-package-command#wp-scaffold-package-tests

I'd encourage you to consider adding functional tests to your package :) By starting your functional tests early on, it also makes it much easier to maintain your project over time.

**Comment by frozzare** [comment]
Yeah, testing is good, finding the time to add testing to all projects is hard.

**Comment by danielbachhuber** [comment]
The scaffolding certainly aims to make writing tests much easier. If you want to give it 5 minutes to use, and let me know if you can't use it in 5 minutes, I'd be very happy to help improve the process.

**Comment by frozzare** [comment]
Will take a look at it later!
